content/id/docs/concepts/overview/object-management-kubectl/imperative-command.md

```diff
@@ -0,0 +1,132 @@
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Kelebihan dan kekurangan
+
+Perintah `kubectl` mendukung tiga cara pengelolaan objek:
+
+* Perintah imperatif
+* Konfigurasi objek imperatif
+* Konfigurasi objek deklaratif
+
+Lihat [Pengelolaan Objek Kubernetes](/id/docs/concepts/overview/object-management-kubectl/overview/) untuk menyimak diskusi mengenai kelebihan dan kekurangan dari tiap cara pengelolaan objek.
+
+## Cara membuat objek
+
+*Tool* `kubectl` mendukung perintah-perintah berbentuk kata kerja untuk membuat beberapa tipe objek yang paling umum. Perintah-perintah tersebut diberi nama yang mudah dikenali oleh pengguna yang belum familiar dengan tipe-tipe objek Kubernetes.
+
+- `run`: Membuat sebuah objek Deployment untuk menjalankan kontainer di satu atau lebih Pod.
+- `expose`: Membuat sebuah objek Service untuk mengatur lalu lintas beban antar Pod.
+- `autoscale`: Membuat sebuah objek Autoscaler untuk melakukan *scaling* horizontal secara otomatis terhadap sebuah objek *controller*, misalnya sebuah objek Deployment.
+
+*Tool* `kubectl` juga mendukung perintah-perintah pembuatan objek yang berdasarkan pada tipe objek. Perintah-perintah ini mendukung lebih banyak tipe objek dan lebih eksplisit tentang intensi mereka. Tapi, perintah-perintah ini memerlukan pengguna untuk memahami tipe dari objek-objek yang hendak mereka buat.
+
+- `create <objecttype> [<subtype>] <instancename>`
+
+Beberapa tipe objek memiliki sub tipe yang bisa kamu spesifikasikan pada perintah `create`. Misalnya, objek Service memiliki beberapa sub tipe seperti ClusterIP, LoadBalancer, dan NodePort. Berikut adalah sebuah contoh cara membuat sebuah Service dengan sub tipe NodePort:
+
+```shell
+kubectl create service nodeport <myservicename>
+```
+
+Pada contoh di atas, perintah `create service nodeport` merupakan sub perintah dari `create service`.
+
+Kamu bisa menggunakan *flag* `-h` untuk mencari argumen-argumen dan paramenter-parameter yang didukung oleh sebuah sub perintah:
+
+```shell
+kubectl create service nodeport -h
+```
+
+## Cara memperbarui objek
+
+Perintah `kubectl` mendukung perintah-perintah berbasis kata kerja untuk beberapa operasi pembaruan yang umum. Perintah-perintah ini diberi nama yang memudahkan pengguna yang belum familiar dengan objek-objek Kubernetes untuk melakukan pembaruan tanpa terlebih dulu mengetahui *field-field* spesifik yang harus diperbarui:
+
+- `scale`: Melakukan *scaling* horizontal terhadap sebuah *controller* untuk menambah atau menghapus Pod dengan memperbarui jumlah replika dari *controller* tersebut.
+- `annotate`: Menambah atau menghapus anotasi sebuah objek.
+- `label`: Menambah atau menghapus label sebuah objek.
+
+Perintah `kubectl` juga mendukung perintah-perintah pembaruan berdasarkan salah satu aspek dari sebuah objek. Untuk tiap tipe objek yang berbeda, memperbarui sebuah aspek tertentu bisa berarti memperbarui sekumpulan *field* yang berbeda pula:
+
+- `set` `<field>`: Memperbarui sebuah satu aspek dari sebuah objek.
```

Comment:
typo: sebuah satu aspek?

Reply:
should be "salah satu aspek". Updated it in my new commit.